dev/travis/js/config.php

```diff
@@ -0,0 +1,360 @@
+<?php
```

Comment:
I was following [docs](https://github.com/magento/devdocs/blob/develop/guides/v2.2/config-guide/cli/config-cli-subcommands-static-view.md#deploy-static-view-files-without-installing-magento-deploy_without_db) in this matter, b/c installing the whole application, just to get statics, it's serious overhead for me.

Maybe simpler option will be to work on source files, rather than processing them through M core, before running tests? It should be also easier for development purposes.